modules/openxBidAdapter.js

```diff
@@ -188,10 +201,15 @@ const OpenxAdapter = function OpenxAdapter() {
       }
     });
 
-    params.callback = 'window.$$PREBID_GLOBAL$$.oxARJResponse';
-    let queryString = buildQueryStringFromParams(params);
-
-    adloader.loadScript(`//${delDomain}/w/1.0/arj?${queryString}`);
+    try {
+      let queryString = buildQueryStringFromParams(params);
+      let url = `//${delDomain}/w/1.0/arj?${queryString}`;
+      ajax.ajax(url, $$PREBID_GLOBAL$$.oxARJResponse, void (0), {
```

Comment:
Now that JSONP is gone, is there any reason for this function to be on the global?

If not, let's remove it.